bin/generate_requirements.sh

```diff
@@ -26,7 +26,7 @@ if [ ! -d "${requirements_dir}" ]; then
   exit 1
 fi
 
-if grep -Fxq "$adapter" "${requirements_dir}/${version}/pyproject.toml"; then
+if grep -Fxq "$adapter" "${requirements_dir}/pyproject.toml"; then
```

Comment:
The `${version}` variable is already included in `${requirements_dir}` above. It was being duplicated here.